_episodes/02-r-dataframes.md

```diff
@@ -17,7 +17,7 @@ In this lesson, we're going to introduce a package called `dplyr`. dplyr takes a
 
 You may not be familiar with dataframes by name, but you may recognize the structure. Dataframes are arranged into rows and columns, not unlike tables in typical spreadsheet format (ex: Excel). In R, they are represented as vectors of vectors: that is, a vector wherein each column is itself a vector. If you are familiar with matrices, or two-dimensional arrays in other languages, the structure of a dataframe will be clear to you.
 
-However, dataframes are not merely vectors- they are a specific type of object with their own functionality, which we will cover in this lesson.
+However, dataframes are not merely vectors- they are a specific type of object with their own functionality, which we will cover in this lesson. As an aside, the [cheat sheets](https://www.rstudio.com/resources/cheatsheets/) for `dplyr` and `readr` may be useful when reviewing this lesson.
```

Comment:
i believe this wording should be up near line 16 where we try ti describe how to use dplyr.

"a useful tool as we begin to explore `dplyr's` functionality are the cheatsheets available on the rstudio website ______link____" --> something similar perhaps?

this is true for the ACT/FACT/GLATOS/MIGRAMAR curricula below as well